gwctl/pkg/resourcediscovery/resourcemodel.go

```diff
@@ -316,6 +350,13 @@ func (rm *ResourceModel) calculateEffectivePolicies() error {
 // Namespace, and Gateway).
 func (rm *ResourceModel) calculateEffectivePoliciesForGateways() error {
 	for _, gatewayNode := range rm.Gateways {
+		// Do not calculate effective policy for the Gateway if the GatewayClass is
+		// incorrect. For now, we only calculate effective policy once the
```

Comment:
What does "incorrect" mean in this context?

Reply:
Sorry, yes that is not clear! I reworded now:

```
// Do not calculate effective policy for the Gateway if the referenced
// GatewayClass does not exist. For now, we only calculate effective policy
// once the references are corrected.
```